src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__container {
+  width: 96px;
+  height: 116px;
+  border-radius: 0.5px;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both, to avoid potential margin collapse issues. Here you have a right margin, which is fine, but ensure consistency throughout your layout.